src/Components/Card.js

```diff
@@ -1,10 +1,20 @@
+import styled from 'styled-components'
+
+
 const Card = () => {
+
+const CardLi = styled.li`
```

Comment:
Its better to move the definition of this styled component out of the Card component (e.g. a few lines up in the same file). Then the styled component is not being newly created with each rendering of the Card component.

Reply:
Thank you! I always make this mistake... Haha